### PR TITLE
Bump config-seed to `5.2.0` and drop Compose V1 support

### PR DESCRIPTION
**Pull request by mserranom**
## Description

- Updates `lib/config-seed/version` to `5.2.0`
- Removes `version` from docker-compose files, dropping support for Compose V1.

## Contributor Agreement

- [x] I confirm I have signed the [Contributor License Agreement](https://github.com/overleaf/overleaf/blob/master/CONTRIBUTING.md#contributor-license-agreement)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 2024-10-22
+### Added
+- Updated default [`version`](https://github.com/overleaf/toolkit/blob/master/lib/config-seed/version) to `5.2.0`.
+- Drop support for Docker Compose v1. [How to switch to Compose V2](https://docs.docker.com/compose/releases/migrate/#how-do-i-switch-to-compose-v2).
+
```
Comment (das7pad) [inline CHANGELOG.md:7]:
```suggestion
  Docker Compose v1 has reached its End Of Life in July 2023.

```
WDYT about adding this as a reminder why we are dropping support?

**Review by das7pad — APPROVED** [review]
:broom: Thanks!

File: CHANGELOG.md (hunk 1 above)
Comment (mserranom) [reply CHANGELOG.md:8]: Good idea.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 2024-10-22
```
Comment (mserranom) [inline CHANGELOG.md:3]:
```suggestion
## 2024-10-24
```

**Pull request merged**